### PR TITLE
Complete review validator. See Issue #13.

### PR DESCRIPTION
**Pull request by nicholasnbg**
As per issue #13

### CODE REVIEW

File: src/main/scala/com/reagroup/appliedscala/urls/savereview/NewReviewValidator.scala [hunk 1]
```diff
@@ -21,7 +21,11 @@ object NewReviewValidator {
     * Hint: You can use `.isEmpty` or `.nonEmpty` on `String`
     */
   private def validateAuthor(author: String): ValidatedNel[ReviewValidationError, String] =
-    ???
+    if(author.nonEmpty) {
```
Comment (wjlow) [inline src/main/scala/com/reagroup/appliedscala/urls/savereview/NewReviewValidator.scala:24]:
```suggestion
    if (author.nonEmpty) {
```

File: src/main/scala/com/reagroup/appliedscala/urls/savereview/NewReviewValidator.scala [hunk 2]
```diff
@@ -30,6 +34,10 @@ object NewReviewValidator {
     * Hint: You can use `.isEmpty` or `.nonEmpty` on `String`
     */
   private def validateComment(comment: String): ValidatedNel[ReviewValidationError, String] =
-    ???
+    if(comment.nonEmpty) {
```
Comment (wjlow) [inline src/main/scala/com/reagroup/appliedscala/urls/savereview/NewReviewValidator.scala:37]:
```suggestion
    if (comment.nonEmpty) {
```

**Review by wjlow — APPROVED** [review]
Nice one!

**Comment by wjlow** [comment]
I raised https://github.com/realestate-com-au/applied-scala/issues/17 so we don't have to discuss formatting issues in PRs anymore.

**Review by millyrowboat — APPROVED** [review]

**Pull request merged**